buildpack-health-check/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/cyberark/conjur-api-go/conjurapi"
+	"github.com/cyberark/conjur-api-go/conjurapi/authn"
+)
+
+func main() {
+	followerURL := os.Getenv("CONJUR_FOLLOWER_URL")
+	if len(followerURL) > 0 {
+		os.Setenv("CONJUR_APPLIANCE_URL", followerURL)
```

Comment:
This is getting set in the service broker environment I believe? Is it okay to leave it around or should we clean it up?

Reply:
You are referring to the `CONJUR_APPLIANCE_URL` environment variable, correct?

No, this only sets it in the context of the `buildpack-health-check` execution, and doesn't change the environment of the calling script, `start-service-broker.sh`. The service broker will still see the `CONJUR_APPLIANCE_URL` as configured in the tile.